packages/atomic/src/components/common/facets/facet-common.tsx

```diff
@@ -342,6 +448,10 @@ export class FacetCommon {
     this.dependenciesManager?.stopWatching();
   }
 
+  private get isHidden() {
+    return !this.facet.state.enabled || !this.facet.state.values.length;
+  }
```

Comment:
Didn't think about this before, but I think `canBeExpanded` would be more relevant/direct and avoids accidentally breaking this in the future.